.editorconfig

```diff
@@ -0,0 +1,110 @@
+# EditorConfig helps developers define and
+# maintain consistent coding styles between
+# different editors and IDEs
+
+# http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+charset = utf-8
+
+[*.{proj,csproj,vcxproj,xproj,json,config,nuspec,xml}]
+indent_size = 2
+
+# Sort using and Import directives with System.* appearing first
+dotnet_sort_system_directives_first = true
```

Comment:
The key `dotnet_sort_system_directives_first` also appear on line 86. I don't think it makes sense here?